### PR TITLE
updated

### PR DESCRIPTION
**Pull request by jdgemm**
privacy and terms docs

### CODE REVIEW

File: sidebarsDocs.js [hunk 1]
```diff
@@ -49,7 +49,9 @@ module.exports = {
           'use-the-network/run-a-network-server/run-a-network-server',
           'use-the-network/setup-a-packet-forwarder',
           'use-the-network/community-projects/community-projects',
-          'use-the-network/coverage-mapping/coverage-mapping'
+          'use-the-network/coverage-mapping/coverage-mapping',
+          'use-the-network/policies'
```
Comment (kent-williams) [inline sidebarsDocs.js]: This doc doesn't exist yet

File: sidebarsDocs.js [hunk 2]
```diff
@@ -198,6 +200,19 @@ module.exports = {
       collapsed: false,
     },
   ],
+  policies: [
+    {
+       type: 'link',
+       label: '<- Back',
+       href: '/use-the-network/policies'
```
Comment (kent-williams) [inline sidebarsDocs.js:207]: This doc doesn't exist yet

File: sidebarsDocs.js [hunk 3]
```diff
@@ -198,6 +200,19 @@ module.exports = {
...
+     },
+     {
+       type: 'category',
+       label: 'Policies',
+       items: ['use-the-network/policies/policies','use-the-network/policies/privacy', 'use-the-network/policies/terms'],
```
Comment (kent-williams) [inline sidebarsDocs.js]: We usually don't put the top level item, `use-the-network/policies/policies`, in the sub menu.

Comment (jdgemm) [reply sidebarsDocs.js]: resolved, thanks

**Review by kent-williams — DISMISSED** [review]

**Review by kent-williams — APPROVED** [review]

**Pull request merged**